test/src/087-xattrs/main

```diff
@@ -117,6 +117,22 @@ cvmfs_run_test() {
   [ "x$proxy_list" != "x" ] || return 21
   [ "x$proxy_list_external" != "x" ] || return 22
 
+  # check different access modes 
+  echo "*** Check different access modes: human vs machine"
+  runc_chunk_list_machine=$(get_xattr chunk_list~0 /cvmfs/grid.cern.ch/vc/containers/runc)
+  runc_chunk_list_human=$(get_xattr chunk_list@0 /cvmfs/grid.cern.ch/vc/containers/runc)
```

Comment:
Also add checks for mistyped attr names: page numbers `-1`, `-2`, and `<string that is not a number>`, and nothing (`chunk_list~`)